### PR TITLE
well list search done.

### PR DESCRIPTION
**Pull request by abhishekk57**
well list search done.

### CODE REVIEW

File: src/app/common-components/sub-header/sub-header.component.html [hunk 1]
```diff
@@ -43,7 +32,15 @@
     placeholder="Search"
     [animated]="true"
     [debounce]="1000"
-  ></ion-searchbar>
+  ></ion-searchbar> -->
+  <div *ngIf="isSearch" class="diva1"
+    style="display: flex; flex: 1;flex-direction: row; align-items: center; height: 58px;">
+    <div (click)="onBackBtnClick()" class="icon-16">
+      <ion-icon style="color: #fff;font-size: 28px;" name="chevron-back-outline"></ion-icon>
```
Comment (gopi2mohan) [inline src/app/common-components/sub-header/sub-header.component.html:39]: don't use inline style

File: src/app/common-components/sub-header/sub-header.component.html [hunk 2]
```diff
@@ -43,7 +32,15 @@
     placeholder="Search"
     [animated]="true"
     [debounce]="1000"
-  ></ion-searchbar>
+  ></ion-searchbar> -->
+  <div *ngIf="isSearch" class="diva1"
+    style="display: flex; flex: 1;flex-direction: row; align-items: center; height: 58px;">
```
Comment (gopi2mohan) [inline src/app/common-components/sub-header/sub-header.component.html:37]: don't use inline style

File: src/app/common-components/sub-header/sub-header.component.html [hunk 3]
```diff
@@ -93,26 +90,15 @@
...
-            "
-            name="cars"
-            id="cars"
-          >
-            <option
-              style="background-color: #535353; width: 90px"
-              value="volvo"
-            >
+            " name="cars" id="cars">
+            <option style="background-color: #535353; width: 90px" value="volvo">
```
Comment (gopi2mohan) [inline src/app/common-components/sub-header/sub-header.component.html:98]: don't use inline style

File: src/app/common-components/sub-header/sub-header.component.html [hunk 4]
```diff
@@ -93,26 +90,15 @@
...
               Select Status Type
             </option>
-            <option
-              style="background-color: #535353; max-width: 50%"
-              value="Optimum Pumping"
-            >
+            <option style="background-color: #535353; max-width: 50%" value="Optimum Pumping">
```
Comment (gopi2mohan) [inline src/app/common-components/sub-header/sub-header.component.html:101]: don't use inline style

File: src/app/common-components/sub-header/sub-header.component.html [hunk 5]
```diff
@@ -93,26 +90,15 @@
...
               Optimum Pumping
             </option>
             <option style="background-color: #535353" value="Under Pumping">
```
Comment (gopi2mohan) [inline src/app/common-components/sub-header/sub-header.component.html:104]: don't use inline style

File: src/app/common-components/sub-header/sub-header.component.html [hunk 6]
```diff
@@ -126,16 +112,10 @@
             </option>
           </select>
           <div style="align-self: flex-end; margin-left: 50%; margin-top: 10%">
```
Comment (gopi2mohan) [inline src/app/common-components/sub-header/sub-header.component.html:114]: don't use inline style

**Review by gopi2mohan — COMMENTED** [review]
please resolve this comment

File: src/app/common-components/sub-header/sub-header.component.html (hunk 6 above)
Comment (abhishekk57) [reply src/app/common-components/sub-header/sub-header.component.html]: fixed.

File: src/app/common-components/sub-header/sub-header.component.html (hunk 5 above)
Comment (abhishekk57) [reply src/app/common-components/sub-header/sub-header.component.html]: fixed.

File: src/app/common-components/sub-header/sub-header.component.html (hunk 4 above)
Comment (abhishekk57) [reply src/app/common-components/sub-header/sub-header.component.html]: fixed.

File: src/app/common-components/sub-header/sub-header.component.html (hunk 3 above)
Comment (abhishekk57) [reply src/app/common-components/sub-header/sub-header.component.html]: fixed.

File: src/app/common-components/sub-header/sub-header.component.html (hunk 2 above)
Comment (abhishekk57) [reply src/app/common-components/sub-header/sub-header.component.html]: fixed.

File: src/app/common-components/sub-header/sub-header.component.html [hunk 7]
```diff
@@ -1,33 +1,22 @@
 <div class="div1">
-  <div class="div2">
-    <p
-      style="font-size: 18px; font-weight: 700; font-family: 'SLBSans'"
-      (click)="toggle()"
-    >
+  <div *ngIf="!isSearch" class="div2">
+    <p style="font-size: 18px; font-weight: 700; font-family: 'SLBSans'" (click)="toggle()">
```
[the remark this answers is not in the document]
Comment (abhishekk57) [reply src/app/common-components/sub-header/sub-header.component.html:3]: fixed.

File: src/app/common-components/sub-header/sub-header.component.html (hunk 1 above)
Comment (abhishekk57) [reply src/app/common-components/sub-header/sub-header.component.html]: fixed.

**Comment by naveensh26** [comment]
calender added for search ,

**Pull request merged**